### PR TITLE
[service.linuxwhatelse.notify] v2.0.0

### PR DESCRIPTION
**Pull request by Tadly**
### Description

Kodi add-on component for my android-app that forwards notifications and call events.

- Notifications will show up on kodi
- Calls can pause the playback or mute/lower the volume

Requires #432

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [sciprt.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by Tadly** [comment]
just force-pushed an update to addon.xml (not sure how that mess slipped through)

File: service.linuxwhatelse.notify/addon/utils.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+addon = xbmcaddon.Addon()
+
+def translate(id):
+    return addon.getLocalizedString(id)
+
+def log(*args, **kwargs):
+    if not kwargs or 'lvl' not in kwargs:
+        lvl = xbmc.LOGNOTICE
```
Comment (enen92) [inline service.linuxwhatelse.notify/addon/utils.py]: Can you move the default log level to degub please?

**Comment by enen92** [comment]
Fanart is required according to repository submission rules

**Comment by enen92** [comment]
Also, the icon cant be a transparent PNG. It has to have some sort of background

**Comment by enen92** [comment]
@Tadly can you also change the language directory tree to something like this: https://github.com/kuzin2006/repo-plugins/tree/ec512bf659dec07f17fd9bf2f7c09369831c2203/plugin.audio.radio.roks/resources/language

Appart from the comments, I can't find anything else to annoy you :) Thanks

**Comment by Tadly** [comment]
Whew! Finally...
I think/hope I got everything

**Comment by enen92** [comment]
crap missed it for 20 seconds :)

**Pull request merged**